### DISCUSSION TITLE
Can standard users install apps?

### DISCUSSION THREAD

**Discussion by tbcahill** [Devolutions/UniGetUI#1698] [Q&A] [answered]
If an admin installs WingetUI or deploys it via an MDM, are users who don't have admin permissions still able to use WingetUI to install app.

My use case is that I'm hoping to remove admin permissions from a set of users & then allow them to install a selection of pre-approved apps via WingetUI

**Comment by Spacellary** [comment, marked as answer]
@tbcahill Installing applications in user space only using the Scoop source won't require Admin privileges. Now if the application needs to be elevated to work properly that's a different problem, but that's more on the application itself.

So WingetUI could be deployed with only Scoop as source. Not all applications will be able to be installed that way though, given some installers (don't enable the `nonportable` bucket) that require UAC are still a thing, but if you stick to the appropriate [buckets](https://github.com/ScoopInstaller/Scoop?tab=readme-ov-file#known-application-buckets) it should work.

**Reply by marticliment** [reply]
@Spacellary is right

**Reply by tbcahill** [reply]
Thanks for the info @Spacellary. I'll play around with Scoop to see if I can get it to work for my use case
